src/main/java/jenkins/plugins/git/traits/RemoteNameSCMSourceTrait.java

```diff
@@ -176,6 +177,7 @@ private static String validate(@NonNull String value) {
      * Our {@link hudson.model.Descriptor}
      */
     @Extension
+    @Symbol("gitRemoteName")
```

Comment:
Would prefer that traits end with the suffix 'Trait' so that they don't prevent using that symbol for an extension that has the same name (like this one does).

```suggestion
    @Symbol("gitRemoteNameTrait")
```